apps/www/_blog/2023-08-11-supavisor-1-million.mdx

```diff
@@ -0,0 +1,436 @@
+
+|        | Query Duration with Supavisor | Query Duration with PgBouncer |
+| ------ | ----------------------------- | ----------------------------- |
+| Median | 4ms                           | 1ms                           |
+| p95    | 4ms                           | 2ms                           |
+| p99    | 5ms                           | 3ms                           |
+
+<div>
+  <img
+    style={{ marginBottom: '0em' }}
+    alt="chart query duration for 5k qps inserts with supavisor light"
+    className="dark:hidden"
+    src="/images/blog/2023-08-11-supavisor-1-million/chart-supavisor-duration-light.png"
+  />
+  <img
+    style={{ marginBottom: '0em' }}
+    alt="chart query duration for 5k qps inserts with supavisor dark"
+    className="hidden dark:block"
+    src="/images/blog/2023-08-11-supavisor-1-million/chart-supavisor-duration-dark.png"
+  />
+  <figcaption>Fig.1 - Query Duration with Supavisor</figcaption>
+</div>
+
+<div>
+  <img
+    style={{ marginBottom: '0em' }}
+    alt="chart query duration for 5k qps inserts with pgbouncer light"
+    className="dark:hidden"
+    src="/images/blog/2023-08-11-supavisor-1-million/chart-pgbouncer-duration-light.png"
+  />
+  <img
+    style={{ marginBottom: '0em' }}
+    alt="chart query duration for 5k qps inserts with pgbouncer dark"
+    className="hidden dark:block"
+    src="/images/blog/2023-08-11-supavisor-1-million/chart-pgbouncer-duration-dark.png"
+  />
+  <figcaption>Fig.2 - Query Duration with PgBouncer</figcaption>
+</div>
+
+### Getting started
+
+Supavisor has been rolled out to all Supabase projects in all regions.
+
+Contact [support](https://www.notion.so/83ee995954214527986a8375156c9e6c?pvs=21) to start using it today, and we'll provide connection details. We will be exposing a new connection string in project dashboards over the next few weeks.
+
+You'll be able to use both PgBouncer and Supavisor for a few months in parallel. Nothing will change with your PgBouncer setup if you need to switch back.
+
+Supavisor will be added to the self-hosted stack as soon as we have tested it across our database fleet. That said - we're confident that it's ready for use if you want to try it with your own Postgres database. [Sequin](https://sequin.io/), one of our partners, has been using Supavisor for several months:
+
+<div className="bg-gray-300 rounded-lg px-6 py-2 italic">
+
+With Supavisor, we've been able to ship incredible features that would have been very hard to build otherwise. For example, our customers can now read from and write to Salesforce and HubSpot via Postgres. We achieve this by intercepting queries that route through Supavisor.
+
+We chose Supavisor because it's scalable, multi-tenant, and written in Elixir. We were able to integrate it easily with our own Elixir infrastructure. As partners, we look forward to helping shape the future of Postgres connection pooling with Supabase.
+
+— Anthony Accomazzo - Co-founder of Sequin
+
+</div>
```

Comment:
We actually have a "Quote" component that we can use here - see it in this blog post:

https://supabase.com/blog/supabase-series-a#about-the-round